moveit_commander/src/moveit_commander/planning_scene_interface.py

```diff
@@ -51,112 +52,53 @@
         pyassimp = False
         print("Failed to import pyassimp, see https://github.com/ros-planning/moveit/issues/86 for more info")
 
-# This is going to have more functionality; (feel free to add some!)
-# This class will include simple Python code for publishing messages for a planning scene
 
 class PlanningSceneInterface(object):
     """ Simple interface to making updates to a planning scene """
 
-    def __init__(self):
+    def __init__(self, synchronous=False, service_timeout=5.0):
```

Comment:
It looks like keeping ``synchronous=False`` is good because of backwards compatibility in ROS Kinetic - we don't want to change behavior. Perhaps add a comment to change this to True in ROS-Melodic?